packages/core/src/cloud/config.ts

```diff
@@ -0,0 +1,74 @@
+import type { PlatformApi } from "@llamaindex/cloud";
+import { BaseNode, TextNode } from "../Node.js";
+import { OpenAIEmbedding } from "../embeddings/OpenAIEmbedding.js";
+import type { TransformComponent } from "../ingestion/types.js";
+import { SimpleNodeParser } from "../nodeParsers/SimpleNodeParser.js";
+
+export type GetPipelineCreateParams = {
+  pipelineName: string;
+  pipelineType: PlatformApi.PipelineType;
+  transformations?: TransformComponent[];
+  inputNodes?: BaseNode[];
+};
+
+function getTransformationConfig(
+  transformation: TransformComponent,
+): PlatformApi.ConfiguredTransformationItem {
+  if (transformation instanceof SimpleNodeParser) {
+    return {
+      configurableTransformationType: "SIMPLE_FILE_NODE_PARSER",
```

Comment:
@himself65 have to use a string constant here as using the value from `PlatformApi.ConfigurableTransformationNames` would require a non-type import of `PlatformApi` which would be converted to a `require` call - any idea how to use this in a type safe manner with our current config?